cfme/infrastructure/virtual_machines.py

```diff
@@ -684,6 +685,28 @@ def active(self):
             else:
                 return has_child(view.tree, '{} (Active)'.format(title), root_item)
 
+        @property
+        def size(self):
+            """
+            Check the shapshot size in the UI. So far available only in RHV and CFME > "5.11"
```

Comment:
Anyway to get this info via a REST query? Or is this designed so that you can test the UI?

Reply:
yes :) the feature is to show this in the UI :)